ReactiveCocoa.playground/Pages/SignalProducer.xcplaygroundpage/Contents.swift

```diff
@@ -360,7 +364,7 @@ scopedExample("`collect(count:)`") {
 }
 
 /*:
- ### ``collect(predicate:)` matching values inclusively`
+ ### `collect(predicate:) matching values inclusively`
```

Comment:
I don't think the fix is correct. This should be ``### `collect(predicate:)` matching values inclusively``.